aws/data_source_aws_rds_cluster.go

```diff
@@ -221,7 +221,7 @@ func dataSourceAwsRdsClusterRead(d *schema.ResourceData, meta interface{}) error
 	// RDS accepts a DatabaseName but does not return it, causing a perpetual
 	// diff.
 	//	See https://github.com/hashicorp/terraform/issues/4671 for backstory
-	if dbc.DatabaseName != nil {
+	if dbc.DatabaseName != nil { // nosem: helper-schema-ResourceData-Set-extraneous-nil-check
```

Comment:
I do wonder if the API bug discussed here has been fixed yet; but for now the safest option is probably just to leave this well alone!